ui/analyse/src/autoShape.ts

```diff
@@ -1,7 +1,7 @@
 import { winningChances } from 'ceval';
-import * as cg from 'chessground/types';
-import { opposite } from 'chessground/util';
-import { DrawModifiers, DrawShape } from 'chessground/draw';
+import * as cg from 'chessground/build/types';
```

Comment:
I updated chessground and stratops to generate files inside a build folder instead of generating files in the root folder of the project
It is only related to that I guess :)